client/reader/controller.js

```diff
@@ -1,16 +1,17 @@
 /**
  * External Dependencies
  */
-var ReactDom = require( 'react-dom' ),
+const ReactDom = require( 'react-dom' ),
 	React = require( 'react' ),
 	page = require( 'page' ),
 	debug = require( 'debug' )( 'calypso:reader:controller' ),
-	trim = require( 'lodash/string/trim' );
+	trim = require( 'lodash/string/trim' ),
+	ReduxProvider = require( 'react-redux' ).Provider;
```

Comment:
I noticed it's called `ReduxProvider` in our other controllers, so I stuck with that:

https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=reduxprovider